examples/library/test_printing/main.c

```
@@ -50,6 +50,7 @@
 #include <time.h>
 #include <stdlib.h>
 #include <bsg_manycore.h>
+#include <bsg_manycore_printing.h>
```

why is this here?

---

Fails silently with a linker error if it isn't.

---

I'll pull it out into a separate PR